optimum/intel/generation/modeling.py

```diff
@@ -93,9 +93,7 @@ class PreTrainedModel(OptimizedModel):
     pass
 
 
-class TSModelForCausalLM(PreTrainedModel, GenerationMixin):
-    auto_model_class = AutoModelForCausalLM
-    export_feature = "text-generation"

```

Comment:
We should keep both `auto_model_class` and `export_feature` no ?

Reply:
done